src/lib/cat_superhero.svelte

```diff
@@ -73,8 +142,44 @@
             />
           </div>
           <div class="text">
-            <h1>{i + 1}. {title}</h1>
+            <div class="title-row">
```

Comment:
This is another nitpick, but since users are on mobile they want to click the entire row to expand the accordion (which would involve moving the event listeners to this element).  Again, that's up to you.